packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -236,26 +238,21 @@ const enableReactRefresh = (config) => {
         },
         entry: {
             ...config.entry,
-            main: [mode === development && 'webpack-hot-middleware/client', './app/main'].filter(
-                Boolean
-            )
+            main: ['webpack-hot-middleware/client?path=/__mrt/hmr', './app/main']
         },
         plugins: [
             ...config.plugins,
 
-            mode === development && new webpack.HotModuleReplacementPlugin(),
-            mode === development &&
-                new ReactRefreshWebpackPlugin({
-                    overlay: {
-                        sockIntegration: 'whm'
-                    }
-                })
-        ].filter(Boolean),
+            new webpack.HotModuleReplacementPlugin(),
+            new ReactRefreshWebpackPlugin({
+                overlay: {
+                    sockIntegration: 'whm'
+                }
+            })
+        ],
         output: {
             ...config.output,
-            // So that client-side hot reloading works properly,
-            // we'll need to prepend the *.hot-update.json urls with this publicPath
-            ...(mode === development ? {publicPath: '/mobify/bundle/development/'} : {})
+            publicPath: '/mobify/bundle/development/'
```

Comment:
I'll need to move this to the client config. We'll always need this for client regardless of hot reloading, since we changed the loadable extractor public path.

Reply:
i disabled react fast refresh and from my testing, everything is still working. so i think we can keep this in `enableReactRefresh`.